modules/cnf-fast-event-notifications-api-refererence.adoc

```diff
@@ -179,7 +179,7 @@ The system generates notifications when the relevant equipment state changes.
 * `os-clock-sync-state` notifications describe the host operating system clock synchronization state. Can be in `LOCKED` or `FREERUN` state.
 * `ptp-clock-class-change` notifications describe the current state of the PTP clock class.
 * `lock-state` notifications describe the current status of the PTP equipment lock state. Can be in `LOCKED`, `HOLDOVER` or `FREERUN` state.
-* `gnss-sync-status` notifications describe the GPS synchronization state with regard to the external GNSS clock signal. Can be in `LOCKED` or `FREERUN` state.
+* `gnss-sync-status` notifications describe the GPS synchronization state with regard to the external GNSS clock signal. Can be in `SYNCHRONIZED` or `FREERUN` state.
```

Comment:
There is no FREERUN state for gnss , use ANTENNA_DISCONNECTED

Reply:
As discussed on Slack, I added the third state as well: 
ANTENNA_DISCONNECTED
SYNCHRONIZED
ACQUIRING_SYNC